code/modules/fallout/areas/area.dm

```diff
@@ -733,6 +733,52 @@
 	environment = 15
 	grow_chance = 0
 
+//Badlands Parent Areas, Start.
+//Power later, when we sort the plant's function out gameplay wise. - Carl
+/area/f13/bl
+	name = "MASTER AREA DO NOT USE"
+	icon_state = "admin"
+	ambientsounds = list('sound/f13ambience/bird_1.ogg','sound/f13ambience/bird_2.ogg','sound/f13ambience/bird_3.ogg','sound/f13ambience/bird_4.ogg','sound/f13ambience/bird_5.ogg','sound/f13ambience/bird_6.ogg','sound/f13ambience/bird_7.ogg','sound/f13ambience/bird_8.ogg')
```

Comment:
I vote for this area to only have honk.ogg but that's just me

Reply:
Flip a coin?